lib/fizzy/execute.cpp

```diff
@@ -212,6 +212,50 @@ Instance instantiate(const Module& module)
     return instance;
 }
 
+namespace
```

Comment:
Damn, these helpers should have been next to the other load/store helpers and not between instantiate/execute.

Reply:
Should I move it in an additional PR?